src/components/ScenarioParameters/ScenarioParameters.js

```diff
@@ -50,32 +51,34 @@ const getRunTemplateParametersIds = (runTemplatesParametersIdsDict, runTemplateI
 
 const ScenarioResetValuesContext = React.createContext();
 
-const ScenarioParameters = ({
-  editMode,
-  changeEditMode,
-  onChangeAccordionSummaryExpanded,
-  accordionSummaryExpanded,
-}) => {
+const ScenarioParameters = ({ onChangeAccordionSummaryExpanded, accordionSummaryExpanded }) => {
   const classes = useStyles();
   const { t } = useTranslation();
   const {
     scenariosData,
     datasetsData,
     addDatasetToStore,
+    currentScenario,
```

Comment:
We use `currentScenario` prop only to get its status, maybe it's better to create a `currentScenarioStatus` hook to avoid getting whole current scenario data twice (in `currentScenario` and `currentScenarioData`) ?

Reply:
I think it is more simple to keep only currentScenario prop and make several const _currentScenarioData_ and _currentScenarioStatus_ from this prop.